autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -64,15 +65,23 @@ const SectionSelect: React.FC<SectionSelectProps> = ({ id }): JSX.Element => {
 
   const makeList = (): JSX.Element[] => {
     let lastProf: string = null;
+    let lastHonors = false;
     return sections.map(({ section, selected, meetings }, secIdx) => {
-      const instructorLabel = lastProf !== section.instructor.name
+      const makeNewGroup = lastProf !== section.instructor.name || lastHonors !== section.honors;
+      const instructorLabel = makeNewGroup
         ? (
           <ListSubheader disableGutters className={styles.listSubheaderDense}>
             {section.instructor.name}
+            {section.honors ? (
+              <Tooltip title="Honors" placement="right">
+                <HonorsIcon titleAccess="honors" />
```

Comment:
I don't think we want this `titleAccess` now that we have an actual tooltip, since now there are two of them:
![image](https://user-images.githubusercontent.com/28655462/83336086-6b676400-a276-11ea-8790-2ad6b2f3731f.png)

Reply:
Unfortunately, I was using the built-in tooltip during testing, so I had to replace that with a `testId`. However, Material's tooltip looks better than the built-in, so I think that this is fine